wizards/openacademy_wizard.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields, api
+
+class Wizard(models.TransientModel):
+    _name = "openacademy.wizard"
+    _description = "This is the description of Wizard model."
+
+    session_id = fields.Many2one("openacademy.session", string="Session")
```

Comment:
- Remove superfluous string
- Add `required=True`, as without this the wizard couldn't work

Reply:
All items done.